src/management/projects.ts

```diff
@@ -36,6 +40,52 @@ export interface ProjectParentResource {
   type: ProjectParentResourceType;
 }
 
+export const PROJECTS_CREATE_QUESTIONS: Question[] = [
+  {
+    type: "input",
+    name: "projectId",
+    default: "",
+    message:
+      "Please specify a unique project id " +
+      `(${clc.yellow("warning")}: cannot be modified afterward) [6-30 characters]:\n`,
+  },
+  {
+    type: "input",
+    name: "displayName",
+    default: "",
+    message: "What would you like to call your project? (defaults to your project ID)",
+  },
+];
+
+export async function createFirebaseProject(
+  projectId: string,
+  options: { displayName?: string; parentResource?: ProjectParentResource }
+): Promise<FirebaseProjectMetadata> {
+  let spinner = ora("Creating Google Cloud Platform project").start();
+  try {
+    await createCloudProject(projectId, options);
+    spinner.succeed();
+
+    spinner = ora("Adding Firebase to Google Cloud project").start();
+    const projectInfo = await addFirebaseToCloudProject(projectId);
+    spinner.succeed();
+
+    logger.info("");
+    logger.info("🎉🎉🎉 Your Firebase project is ready! 🎉🎉🎉");
```

Comment:
You need to omit emoji if the user is on Windows.

Reply:
Replace the emoji with `=` if `process.platform === win32`